File: modified_scripts/r/merged_scripts/merged_fibonacci.r

# qwen2_5-coder_3b CODE

fibonacci <- function(n) {
  if (n <= 0) return(0)
  if (n == 1) return(1)
  
  prev1 <- 0
  prev2 <- 1
  current <- 0
  
  for (i in 2:n) {
    current <- prev1 + prev2
    prev1 <- prev2
    prev2 <- current
  }
  
  return(current)
}
# gemma_2b CODE

fib_1 <- 0
fib_2 <- 1


while (fib_2 < 100) {

  fib_next <- fib_1 + fib_2
  

  fib_1 <- fib_2
  fib_2 <- fib_next
}
# qwen3_8b CODE
fib <- function(n) {
  if (n == 0) return(0)
  if (n == 1) return(1)
  fib(n - 1) + fib(n - 2)
}
# deepseek-coder_1_3b CODE

# deepseek-coder_6_7b CODE

fibonacci <- function(n) {
    if (n <= 0) return("Input should be a positive integer.")
    else if (n == 1) return(0)
    else if (n == 2) return(1)
    else {
        a = 0; b = 1; c = NULL
        for(i in 3:n){
            c = a + b
            a = b
            b = c
        }
        return(c)
    }
}

# qwen2_5-coder_7b CODE

fibonacci <- function(n){
  if(n <= 1) return(n)
  a <- 0
  b <- 1
  for(i in 2:n){
    temp <- a + b
    a <- b
    b <- temp
  }
  return(b)
}
# gemma_7b CODE

fib <- function(n) {
  if (n < 2) {
    return(n)
  } else {
    return(fib(n-1) + fib(n-2))
  }
}

# qwen3_1_7b CODE

File: modified_scripts/java/merged_scripts/merged_fibonacci.java

// deepseek-coder_1_3b CODE

public class Fibonacci {
     public static int fib(int n){ 
         if(n <= 2 ){ 
             n--;
             return ((3 + (-1 * fib((int)(Math.round(n / 2))))) % Mod);
         }  else { 
             int[] storePrevFive = new int[]{((3 + (-1 * fib(4)) % Mod)), ((6 - (*(-5 / Math.round(& n) ) ^ & mod))) };
             return  storePrevFive[0] + ~storePrevFive[1];
        if(true) System . out , " Hello World ";
     public static void main(String[] args){System.out . println(" Hello World ");}}  

// qwen3_1_7b CODE
public class Fibonacci {
    static int fib(int n) {
        if (n <= 1) {
            return n;
        }
        return fib(n - 1) + fib(n - 2);
    }
}
// qwen3_8b CODE
public class Fibonacci {
    public static void main(String[] args) {
        int n = 10;
        int a = 0, b = 1;
        for (int i = 2; i <= n; i++) {
            int c = a + b;
            a = b;
            b = c;
        }
    }
}
// gemma_7b CODE

public class Fibonacci {

    public int calculate(int n) {
        if (n < 2) {
            return n;
        }
        return calculate(n - 1) + calculate(n - 2);
    }

}

// qwen2_5-coder_3b CODE

public class Fibonacci {
    public static long fib(int n) {
        if (n <= 1) return n;
        long a = 0, b = 1, c;
        for (int i = 2; i <= n; i++) {
            c = a + b;
            a = b;
            b = c;
        }
        return b;
    }
}

// qwen2_5-coder_7b CODE

public class Fibonacci {
    public static void main(String[] args) {
        int n = 10;
        int a = 0;
        int b = 1;
        for (int i = 1; i <= n; ++i) {
            System.out.print(a + " ");
            int sum = a + b;
            a = b;
            b = sum;
        }
    }
}

// deepseek-coder_6_7b CODE

public class Main {
    
    static int fibonacci(int n) {
        if (n == 0) return 0;
        else if (n == 1 || n == 2) return 1;
        else return fibonacci(n - 1) + fibonacci(n - 2);
    }
}

// gemma_2b CODE

public class Fibonacci {

    public static int fibonacci(int n) {
        if (n <= 1) {
            return n;
        }
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}
